packages/wrangler/src/__tests__/secret.test.ts

```diff
@@ -95,6 +96,83 @@ describe("wrangler secret", () => {
 				expect(std.err).toMatchInlineSnapshot(`""`);
 			});
 
+			it("should create secret:bulk", async () => {
+				writeFileSync(
+					"secret.json",
+					JSON.stringify({
+						"secret-name-1": "secret_text",
+						"secret-name-2": "secret_text",
+					})
+				);
+
+				// User counter to pass different secrets to the request mock
+				const counter = 0;
+				setMockResponse(
+					`/accounts/:accountId/workers/scripts/:scriptName/secrets`,
+					"PUT",
+					([_url, accountId]) => {
+						expect(accountId).toEqual("some-account-id");
+						counter + 1;
+
+						return { name: `secret-name-${counter}`, type: "secret_text" };
+					}
+				);
+
+				await runWrangler("secret:bulk ./secret.json --name script-name");
+
+				expect(std.out).toMatchInlineSnapshot(`
+			"🌀 Creating the secrets for the Worker \\"script-name\\"
+			✨ Successfully created secret for key: secret-name-1
+			✨ Successfully created secret for key: secret-name-2
+			✨ Successfully created secrets"
+		`);
+				expect(std.err).toMatchInlineSnapshot(`""`);
+			});
+
+			it("should handle failed secret:bulk", async () => {
+				writeFileSync(
+					"secret.json",
+					JSON.stringify({
+						"secret-name-1": "secret_text",
+						"secret-name-2": "secret_text",
+					})
+				);
+
+				// User counter to pass different secrets to the request mock
+				const counter = 0;
```

Comment:
This should be `let` not `const`, as above.

Reply:
It can be const, which is why the test is passing.